Makefile

```diff
@@ -9,7 +9,7 @@ dummyusers:
 	cd contentcuration/ && python manage.py loaddata contentcuration/fixtures/admin_user_token.json
 
 prodceleryworkers:
-	cd contentcuration/ && celery -A contentcuration worker -l info
+	yarn run celery
```

Comment:
Nice! Good idea simplifying this.

Reply:
I actually realized, it's best to make sure anything in production doesn't depend on yarn -- soon we'll be separating the yarn dependency from the actual runtime dependencies, so it's best to anticipate them here.